acquia/acquia_cms
# ACMS-1840: Configure low code

**Motivation**
Added sitestudio_config_management module.

**Proposed changes**
- Create a new module: sitestudio_config_management to manage Site Studio configurations.

**Alternatives considered**
N/A

**Testing steps**
- Export site studio keys.
- Install Site by running command: 
```./vendor/bin/acms site:install```
- Enable `sitestudio_config_management ` module by running command: 
```./vendor/bin/drush en sitestudio_config_management --yes ```
- In `settings.php`, make sure you've set the Site Studio sync directory path. Ex:
```$settings['site_studio_sync'] = '../config/default/sitestudio';```
- Export the Drupal configurations and you would see Site Studio configurations are exported as well:
```./vendor/bin/drush cex --yes ```
- Now install the site from existing configurations by running below commands:
```
./vendor/bin/drush cset system.site uuid 2a583ede-0912-44ca-bcd7-4f45c4f9882a --yes
./vendor/bin/drush cex --yes // You should see Site Studio configurations exported as well.

./vendor/bin/drush site:install minimal --yes
./vendor/bin/drush cset system.site uuid 2a583ede-0912-44ca-bcd7-4f45c4f9882a --yes
 
./vendor/bin/drush cim --yes // You should see Site Studio configurations imported as well.
```

**Merge requirements**
